### PR TITLE
Feature/metrics fix

### PR DESCRIPTION
**Pull request by justinjj24**
- Metric process pct_q30_bases removed
- Metric process yield_bp_q30 added
- .dockstore.yml added
- docsrc updated by running ./build.sh
- files updated
   - bin/compile_metrics.py
   - bin/metrics/metrics.py 
   - /tests/*
   - nextflow.config (consensus version value in dockerhub and github release '0.7.0' & keep_workdir = false)
   - conf/resources.config (autosomes_non_gap_regions file path)

### CODE REVIEW

**Comment by justinjj24** [comment]
@mhebrard and @nicolas-bertin, along with fixing the metric (ie count instead of pct), I have also changed how the autosome bed file is being pointed to (see commit message for details). This latest addition has been tested locally but not via nextflowtower (ie dockstore)

**Comment by justinjj24** [comment]
Also validated with test run of `NA12878-chr14-AKT1` using the `Nextflow Tower`via `dockstore`

File: .dockstore.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  - name: NPM-sample-qc
+    subclass: nfl
+    primaryDescriptorPath: /nextflow.config
+    publish: True
+    authors:
+      - name: Justin Jeyakani
+        email: gnanakkan@gis.a-star.edu.sg
+        orcid: 0000-0002-1788-0417
+      - name: Mar Gonzalez-Porta
+        email: mar@nalagenetics.com
```
Comment (mhebrard) [inline .dockstore.yml:13]:
```suggestion
        email: mar@nalagenetics.com
        orcid: 0000-0002-1661-7254
```
To add the orcid of Mar

Comment (justinjj24) [reply .dockstore.yml:13]: Mar's ORCID updated.

**Comment by justinjj24** [comment]
The [autosomes_non_gap_regions.bed](https://github.com/c-BIG/NPM-sample-qc/blob/feature/metrics-fix/resources/autosomes_non_gap_regions.bed) file path in the `conf/resources.config` updated with nextflow workflow properties `projectDir` instead giving the repository raw path to the file [9f7c4c0](https://github.com/c-BIG/NPM-sample-qc/pull/69/commits/9f7c4c0d7a2149401e6c67bdeb14c092f7edfeed)

**Review by nicolas-bertin — APPROVED** [review]
good to go

**Pull request merged**